### ISSUE TITLE
Error opening file

### ISSUE THREAD

**Issue by zhulf0804** [wkentaro/labelme#299]
Label in windows and open in mac, error happens: 

Error opening file
'utf8' codec can't decode byte 0xb3 in position 0: invalid start byte

How to resolve the above problem?
thx

**Comment by paidaxingNUDT** [comment]
Do you have any solutions?

**Comment by northeastsquare** [comment]
labelme in windows is gb2312 encoding;
while in ubuntu , its ut-8

**Comment by jjjjohnson** [comment]
@northeastsquare Is there any solution for using both gb2312 and utf-8?

**Comment by Ruolingdeng** [comment]
@jjjjohnson @northeastsquare Have you solved the problem? I met with the same problem. I annotated my images using labelme software in windows system. And now I want to use the json file to do training in Ubuntu 16.04, but it occurred "FileNotFoundError: [Errno 2] No such file or directory: 'datasets/coco/train2017/..\\ResizeImage\\000010.jpg'." When I used the labelme software in Ubuntu system to open the json file, it occurred "make sure /home/xbd/000000.json is a valid label file" Could anybody help me, please? I will be very appreciated with you.

**Comment by TomHeaven** [comment]
I find a workaround: Open `label_file.py` and change line 75 to
```python3
 #with open(filename, 'rb' if PY2 else 'r') as f:
with open(filename, encoding='gb2312') as f: # To read windows labels.
```
The error is caused by different default encoding used by Windows and Linux/Mac OS when opening a file. I'll offer a PR to fix that later.

**Comment by TomHeaven** [comment]
This issue should be fixed by PR https://github.com/wkentaro/labelme/pull/676.

**Comment by Bjoux2** [comment]
> This issue should be fixed by PR #676.
withi this:
 #with open(filename, 'rb' if PY2 else 'r') as f:
with open(filename, encoding='gb2312') as f: # To read windows labels.

I got the result:

Error opening file
Open() got an unexpected keyword argument 'encoding'
